src/puppetlabs/trapperkeeper/bootstrap.clj

```diff
@@ -8,16 +8,20 @@
             [puppetlabs.trapperkeeper.services :as services]
             [puppetlabs.trapperkeeper.common :as common]
             [schema.core :as schema]
-            [me.raynes.fs :as fs]))
+            [me.raynes.fs :as fs]
+            [slingshot.slingshot :refer [try+ throw+]]))
 
 ;; Schemas
-(def BootstrapLine {:namespace schema/Str :service-name schema/Str})
+(def ParsedBootstrapEntry {:namespace schema/Str :service-name schema/Str})
 (def BootstrapFiles [(schema/protocol io/IOFactory)])
+(def AnnotatedBootstrapEntry {:entry schema/Str
+                    :bootstrap-file schema/Str
+                    :line-number schema/Int})
 
 ;; Constants
 (def bootstrap-config-file-name "bootstrap.cfg")
 
-(schema/defn ^:private parse-bootstrap-line! :- BootstrapLine
```

Comment:
I'm fine with removing the `^:private` metadata, but we should probably add some of the big public/private pragma blocks to this file to make it super obvious what we are considering as API.  Especially since you've added a bunch of little helper functions; I think there should probably only be two or three fns that are 'public' in this namespace.